[IMP] sign: secured authentication info added (#3430)
odoo/documentation: content/applications/finance/sign.rst

Choose a :guilabel:`Role Name`, add an :guilabel:`Extra Authentication Step` to confirm the
identity of the person, and if the document can be reassigned to another contact then you can select
:guilabel:`Change Authorized` for the role. A :guilabel:`Color` can also be chosen for the role.
This color can help understand which roles are responsible for which field when configuring a
template.

Secured identification
----------------------

As the owner of a document, you may require an :guilabel:`Extra Authentication Step` through an SMS
verification or via Itsme® (available in Belgium and the Netherlands). Both authentication options
require :ref:`credits <iap/buying_credits>`.

.. seealso::
   - :doc:`In-App Purchase (IAP) <../general/in_app_purchase>`
   - :doc:`SMS pricing and FAQ <../marketing/sms_marketing/pricing/pricing_and_faq>`

SMS verification
~~~~~~~~~~~~~~~~

Go to :menuselection:`Sign --> Configuration --> Roles`. Click in the
:guilabel:`Extra Authentication Step` column for the role, and select :guilabel:`Unique Code Via
SMS`.

.. image:: sign/sms-verification.png
   :align: center
   :alt: Add a hash to your document

> Go to the document to sign, add the :guilabel:`Signature` field for example, and click
> :guilabel:`Send`. A page pops up, select the customer, and click :guilabel:`Send`.

Reviewer: Not sure about "previous", but here's what I'd suggest
```suggestion
Go to the document to sign and add a field that has the previous role attributed, such as the
:guilabel:`Signature` field, and click :guilabel:`Send`. A page pops up, select the customer, and
click :guilabel:`Send`.
```

The person signing the document fills in the :guilabel:`Signature` field and clicks
:guilabel:`Validate & Send Completed Document`. A :guilabel:`Final Validation` page pops up where to
add their phone number. One-time codes are sent by SMS.

.. image:: sign/final-validation.png
   :align: center
   :alt: fill in your phone number for final validation

.. note::
   - This feature is enabled by default.
   - As soon as the :guilabel:`Extra Authentication Step` applies to a role, this validation step is
     requested for any field assigned to this role.

Itsme®
~~~~~~

> Go to :menuselection:`Sign --> Configuration --> Settings` and enable
> :guilabel:`Identify with itsme®`.

Reviewer: ```suggestion
Go to :menuselection:`Sign --> Configuration --> Settings` and enable :guilabel:`Identify with
itsme®`.
```

> Then, go to :menuselection:`Sign --> Configuration --> Roles`. Click in the
> :guilabel:`Extra Authentication Step` column for the role, and select :guilabel:`Via itsme®`.

Reviewer: ```suggestion
Then, go to :menuselection:`Sign --> Configuration --> Roles`. Click in the :guilabel:`Extra
Authentication Step` column for the role, and select :guilabel:`Via itsme®`.
```

> Go to the document that needs to be signed, add the :guilabel:`Signature` field, and select the role
> :guilabel:`customer (identified with itsme®)`, and click :guilabel:`Validate` and :guilabel:`Send`.

Reviewer: That sentence contains a lot of "and" here's what I'd suggest:
```suggestion
Go to the document that needs to be signed and add the :guilabel:`Signature` field. Switch the role
to :guilabel:`customer (identified with itsme®)`, click :guilabel:`Validate`, and :guilabel:`Send`.
```

.. image:: sign/itsme-identification.png
   :align: center
   :alt: select customer identified with itsme®

> The person signing the document fills in the :guilabel:`Signature` field and then click

Reviewer: ```suggestion
The person signing the document fills in the :guilabel:`Signature` field and clicks
```
